app/src/main/java/com/movies/presentation/home/custom_view/NavigationViewItem.kt

```diff
@@ -0,0 +1,52 @@
+package com.movies.presentation.home.custom_view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.FrameLayout
+import androidx.core.content.ContextCompat
+import com.movies.R
+import com.movies.databinding.NavigationCustomViewItemBinding
+
+class NavigationViewItem @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttrs: Int = 0,
+    defStyleRes: Int = 0
+) : FrameLayout(context, attrs, defStyleAttrs, defStyleRes) {
+
+    private val binding =
+        NavigationCustomViewItemBinding.inflate(LayoutInflater.from(context), this, true)
+
+    fun setActiveButton(isInFocus: Boolean) {
```

Comment:
Use more understandable name for function and parameter

Reply:
done